### PR TITLE
[SimplePing] Added the v1.4 component

### PR DESCRIPTION
**Pull request by mattleibow**

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -259,6 +259,10 @@
   BuildScript: ./iOS/SCCatWaitingHUD/build.cake
   TriggerPaths: [ iOS/SCCatWaitingHUD ]
   MacBuildTargets: [ samples, component ]
+- Name: SimplePing
+  BuildScript: ./iOS/SimplePing/build.cake
+  TriggerPaths: [ iOS/SimplePing ]
+  MacBuildTargets: [ component, samples ]
```
Comment (bholmes) [inline manifest.yaml]: Should we reverse the target order to match the others?

` MacBuildTargets: [ samples, component ]`

File: iOS/SimplePing/build.cake [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	NuGets = new [] {
+		new NuGetInfo { NuSpec = "./nuget/Xamarin.SimplePing.nuspec" },
+	},
+
+	Components = new [] {
+		new Component { ManifestDirectory = "./component" },
+	},
+};
+
+Task ("externals")
```
Comment (bholmes) [inline iOS/SimplePing/build.cake:31]: externals folder is missing.  Can you please add it?

File: iOS/SimplePing/source/Xamarin.SimplePing/ApiDefinition.cs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+	// @protocol SimplePingDelegate <NSObject>
+	[Protocol, Model]
+	[BaseType(typeof(NSObject))]
+	interface SimplePingDelegate
+	{
+		// @optional -(void)simplePing:(SimplePing * _Nonnull)pinger didStartWithAddress:(NSData * _Nonnull)address;
+		[Export("simplePing:didStartWithAddress:")]
+		[EventArgs("SimplePingStarted")]
+		[EventName("Started")]
+		void DidStartWithAddress(SimplePing pinger, NSData address);
```
Comment (bholmes) [inline iOS/SimplePing/source/Xamarin.SimplePing/ApiDefinition.cs]: Should this be DidStart?  (suggestion)

File: iOS/SimplePing/source/Xamarin.SimplePing/ApiDefinition.cs [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+		// @optional -(void)simplePing:(SimplePing * _Nonnull)pinger didFailWithError:(NSError * _Nonnull)error;
+		[Export("simplePing:didFailWithError:")]
+		[EventArgs("SimplePingFailed")]
+		[EventName("Failed")]
+		void DidFailWithError(SimplePing pinger, NSError error);
```
Comment (bholmes) [inline iOS/SimplePing/source/Xamarin.SimplePing/ApiDefinition.cs]: Should this be DidFail? (suggestion)

File: iOS/SimplePing/source/Xamarin.SimplePing/ApiDefinition.cs (hunk 3 above)
Comment (bholmes) [reply iOS/SimplePing/source/Xamarin.SimplePing/ApiDefinition.cs]: If changed, review your documentation.

File: iOS/SimplePing/source/Xamarin.SimplePing/ApiDefinition.cs (hunk 4 above)
Comment (bholmes) [reply iOS/SimplePing/source/Xamarin.SimplePing/ApiDefinition.cs]: If changed, review your documentation.

**Review by bholmes — CHANGES_REQUESTED** [review]
I need the external files to finish the review.

File: manifest.yaml (hunk 1 above)
Comment (mattleibow) [reply manifest.yaml]: changed

File: iOS/SimplePing/build.cake (hunk 2 above)
Comment (mattleibow) [reply iOS/SimplePing/build.cake:31]: done.

File: iOS/SimplePing/source/Xamarin.SimplePing/ApiDefinition.cs (hunk 4 above)
Comment (mattleibow) [reply iOS/SimplePing/source/Xamarin.SimplePing/ApiDefinition.cs]: changed.

File: iOS/SimplePing/source/Xamarin.SimplePing/ApiDefinition.cs (hunk 3 above)
Comment (mattleibow) [reply iOS/SimplePing/source/Xamarin.SimplePing/ApiDefinition.cs]: changed.

**Review by bholmes — APPROVED** [review]

**Comment by jamesmontemagno** [comment]
Is this puppy going up on NuGet?

**Comment by mattleibow** [comment]
@jamesmontemagno Yes. We are waiting on a review.

**Comment by bholmes** [comment]
https://www.nuget.org/packages/Xamarin.SimplePing/

**Pull request merged**